example/advection/parthinput.advection

```diff
@@ -34,25 +34,30 @@ x2max = 0.5
 ix2_bc = periodic
 ox2_bc = periodic
 
-nx3 = 1
+nx3 = 64
 x3min = -0.5
 x3max = 0.5
+ix3_bc = periodic
+ox3_bc = periodic
 
 <parthenon/meshblock>
 nx1 = 16
 nx2 = 16
-#nx3 = 1
+nx3 = 16
 
 <parthenon/time>
 tlim = 1.0
-integrator = rk2
+integrator = rk1
```

Comment:
Since the spatial stencil is 1st-order, I would expect 1st-order convergence with `rk2` as well as `rk1`. Does something go wrong?

Reply:
Using rk2 results in worse errors (30-40%) for 1D tests and doesn't converge for the diagonal advection (error only goes down by a factor or 1.4 when doubling linear resolution).
Not entirely sure what's going on (potentially double spatial reconstruction per timestep for rk2 amplifies the first order spatial error), but given that I mainly added to to ensure that the mesh infrastructure works I'm not that concerned about rk2.

Reply:
Hmm interesting. Yes let's leave this for now. It might be worth digging into this later, though.